src/Microsoft.AspNetCore.Authentication.Cookies/Microsoft.AspNetCore.Authentication.Cookies.csproj

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <Description>ASP.NET Core middleware that enables an application to use cookie based authentication.</Description>
-    <TargetFramework>netstandard1.3</TargetFramework>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
That's insane. Not being able to use the ASP.NET Core packages on .NET Desktop will be blocking for many people. Is this change discussed/announced somewhere? Why not cross-compiling the few packages that could take advantage of `netcoreapp2.0` instead of retargeting everything?